sdk/core/azure-core/CMakeLists.txt

```diff
@@ -23,13 +23,30 @@ find_package(Threads REQUIRED)
 
 # min version for `CURLSSLOPT_NO_REVOKE`
 # https://curl.haxx.se/libcurl/c/CURLOPT_SSL_OPTIONS.html
-set(CURL_MIN_REQUIRED_VERSION 7.44)
+set(CURL_MIN_REQUIRED_VERSION 7.4)
```

Comment:
Based on offline discussion, we don't want to set the minimum required to a version from 2000, which is too old for the distros we support anyway (like Ubuntu 16.04).
https://curl.se/docs/releases.html

Leave it as 7.44, and let's re-evaluate if we should bump it up even higher.